spring-pulsar/src/main/java/org/springframework/pulsar/listener/DefaultPulsarMessageListenerContainer.java

```diff
@@ -239,6 +240,10 @@ private void populateAllNecessaryPropertiesIfNeedBe(Map<String, Object> currentP
 			if (negativeAckRedeliveryBackoff != null) {
 				currentProperties.put("negativeAckRedeliveryBackoff", negativeAckRedeliveryBackoff);
```

Comment:
**[UNRELATED]** Not directly related to this PR as it came to mind during the previous similar one.... these constants are starting to proliferate (across classes as well). Not sure where a good home would be for them as I do like having the constants "close to the code" but just something for us to all noodle on as we go back in here next time etc..

Reply:
Created an issue to track this: https://github.com/spring-projects-experimental/spring-pulsar/issues/90